terraform/modules/loadbalancer/security_groups.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_security_group" "lb" {
+  vpc_id = var.vpc.aws_vpc.id
+  tags   = merge(var.tags, { Name = "${var.name}-lb" })
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_security_group_rule" "lb_external_https_in" {
+  description       = "enable inbound connectivity from whitelisted endpoints"
+  from_port         = 443
+  protocol          = "tcp"
+  security_group_id = aws_security_group.lb.id
+  to_port           = 443
+  type              = "ingress"
+  cidr_blocks       = var.whitelist_cidr_blocks
+}
```

Comment:
What's this one for?  We're only interested in `9090` ?

Reply:
That's ingress to the loadbalancer to allow https requests. The user would not go directly to 9090